src/core/src/sys/compute.rs

```diff
@@ -236,8 +236,8 @@ impl Compute for PressureAtTemperature {
         let virial_tensor = system.virial();
         let virial = virial_tensor.trace();
         let volume = system.volume();
-        let natoms = system.size() as f64;
-        return natoms * K_BOLTZMANN * self.temperature / volume + virial / (3.0 * volume);
+        let nmols = system.molecules().len() as f64;
```

Comment:
> Does this mean that computing the pressure by using the number of atoms will yield the wrong result?

Yes, I think so, but let me double check tomorrow as well. :) FWIW I got the right pressure (i.e. external == internal) performing the simulation with the number of molecules instead of atoms.